source/general/pkteam.rst

```diff
@@ -8,120 +8,138 @@ The ProjectKorra team is comprised of volunteers who work on various aspects of
 
 Meet the Team
 =============
+
+Administrators
+--------------
+Admins are responsible for the day to day operations of ProjectKorra.
+
 +-----------------+-----------------+---------------+
 | Member          | Position        | MC Avatar     |
 +-----------------+-----------------+---------------+
-| Mist            | Admin & Founder | |mist|        |
+| MistPhizzle     | Admin & Founder | |mist|        |
 +-----------------+-----------------+---------------+
 
-Developers
-----------
 
-Developers are the people who maintain and work on the various
-development projects that require programming knowledge
+Dev Team
```

Comment:
All instances of `Dev` should be replaced with `Development Team` or `Developers`.